internal/schemautil/custom_diff.go

```diff
@@ -116,6 +116,16 @@ func CustomizeDiffCheckPlanAndStaticIpsCannotBeModifiedTogether(_ context.Contex
 	return nil
 }
 
+// CustomizeDiffCheckStaticIpsExistence prevents users from modifying `aiven_static_ip` resources in
+// the same step with the resources that uses them due to Terraform limitation
+// See: https://terraform.io/plugin/sdkv2/resources/customizing-differences
+func CustomizeDiffCheckStaticIpsExistence(_ context.Context, d *schema.ResourceDiff, _ interface{}) error {
```

Comment:
Do I get this right? 

Do we error when a service static_ip's list is empty, and a resource already exists? In this case, what would happen if a customer doesn't care about the static IPs and updates his service with something unrelated? I guess a user would get this error in this case.

Reply:
we error in the following scenario:

- there is already a service deployed via Terraform/that Terraform is aware of
- the service has `aiven_static_ip` dependence, i.e. the service has static IPs assigned to it
- user tries to update `aiven_static_ip` resource and the dependent service in one `terraform apply` run